pkg/apiserver/apiserver.go

```diff
@@ -130,7 +130,7 @@ type APIServer struct {
 	OpenpitrixClient openpitrix.Client
 
 	//
-	LoggingClient logging.Interface
+	LoggingClient logging.Client
```

Comment:
Why changing to Client? Interface is useful for unit testing.

Reply:
The `client` is more accurate for describing the function of this interface, which is conducive to improving code readability.

Reply:
logging.Client is still an interface, just the interface name is changed from Interface to Client